wg-code-of-conduct/code_of_conduct.md

```diff
@@ -53,18 +55,19 @@ decisions when appropriate.
 This Code of Conduct applies within all community spaces, and also applies when
 an individual could reasonably be perceived as representing the community in 
 public spaces. Examples of representing our community include using an 
-apache.org email address, posting via an official social media account, or 
-acting as an appointed representative at an online or offline event.
+apache.org email address, posting via an official Apache social media account, or 
+acting as an appointed Apache or [PROJECT] representative at an online or offline event.
 
 ## Enforcement
 
```

Comment:
Do we want to consider the case of the unacceptable behaviour being from an ASF or PMC member who can see the private list?